plugin-registration.md

```diff
@@ -121,9 +121,9 @@ Key | Description
 **name** | the plugin name, required.
 **description** | the plugin description, required.
 **author** | the plugin author name, required.
-**icon** | a name of the plugin icon. The full list of available icons can be found in the [UI documentation](../ui/icon). Any icon names provided by this font are valid, for example **icon-glass**, **icon-music**.
-**iconSvg** | an SVG icon to be used in place of the standard icon, optional. The SVG icon should be a rectangle and can support colors.
-**homepage** | A link to the author's website address, optional.
+**icon** | a name of the plugin icon. The full list of available icons can be found in the [UI documentation](../ui/icon). Any icon names provided by this font are valid, for example **icon-glass**, **icon-music**, required.
```

Comment:
Is this value actually required? It's only used in one small section in the backend, the plugin details page.

Reply:
will change it to optional.

Reply:
https://github.com/octobercms/docs/pull/468/commits/857e97dad12bd2b6b0b259cec5f946801d4b868f